ContentModification/UnderlinesAndHighlights/UnderlinesAndHighlights.cs

```diff
@@ -7,8 +7,8 @@
  *
  * This program shows how to add annotations to an existing PDF file that will highlight and underline words.
  * When you run it, the program generates a PDF output file. The output sample annotates a PDF file showing
- * a National Weather Service web page, highlighting the word �Cloudy� wherever it appears and underlining
- * the word �Rain.�
+ * a National Weather Service web page, highlighting the word �Cloudy� wherever it appears and underlining
+ * the word �Rain.�
```

Comment:
It's only a minor thing, but it looks like some quotation marks got autocorrected from "plain" ones to "balanced pairs", probably by an overzealous editing program. For code samples, we probably just want the plain ASCII quotes.

Reply:
Fixed